config/tgl-cavs.toml

```diff
@@ -266,3 +266,58 @@ count = 9
 		20, 0, 0, 0, 12832, 20881000, 0, 0, 0, 20881, 0,
 		21, 0, 0, 0, 12832, 23431000, 0, 0, 0, 23431, 0,
 		22, 0, 0, 0, 12832, 30471000, 0, 0, 0, 30471, 0]
+
```

Comment:
thsi conflicts with https://github.com/thesofproject/rimage/pull/120

@libinyang and @aiChaoSONG can't you guys work together?

Reply:
@plbossart I will refine the patch after https://github.com/thesofproject/rimage/pull/120 is merged.